src/components/TableProperty/TableProperty.tsx

```diff
@@ -164,45 +159,56 @@ const TableProperty = () => {
     setProperties(res.data)
   }
 
-  const handleUpdateStatus = async (status:boolean, propertyId: number) => {
-    setStatusProperty(!status)
-    try{Swal.fire({
-      title: 'Are you sure?',
-      text: "You won't be able to revert this!",
-      icon: 'warning',
-      showCancelButton: true,
-      confirmButtonColor: '#3085d6',
-      cancelButtonColor: '#d33',
-      confirmButtonText: 'Yes, disable it!',
-    }).then(async (result) =>{
-      if (result.isConfirmed) {
-        const res = await axiosInstance.patch(`/seller/properties/${propertyId}`, {status: statusProperty})
-        Swal.fire({
-          title: 'Updated!',
-          text: 'Your property is updated.',
-          icon: 'success',
-        })
-        setIsUpdated(prev => !prev)
-        return res
-      }
-    })
-      
-    }catch(error:any) {
-      if (error.response.status === CODE_RESPONSE_400) {
-        Swal.fire({
-          title: 'Failed to update property',
-          icon: 'error',
-        })
-      } else if (error.response.status === CODE_RESPONSE_404) {
-        Swal.fire({
-          title:
-            'This property is not available now. Please try another property!',
-          icon: 'error',
-        })
-      } else {
-        console.error('Error updating property:', error)
-      }
-    }
+  const [checked, setChecked] = useState(false)
+  console.log(checked)
+
+  // const handleUpdateStatus = async (status: boolean, propertyId: number) => {
```

Comment:
If this code is not used, it should be deleted, If you reuse it, you should comment on the purpose this code on top